workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/IComponentReader.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Improbable.Gdk.Core
+{
+    public interface IComponentReader
```

Comment:
I just noticed I left this in, this should not actually exist like this. We'll probably have some supertype once we implement the readers/writers, but the component ID needs to be in the type attribute, because we need access to it when we have a reference to the Type object, not an instance of the type. Probably unnecessary to have this property.